### PR TITLE
feat: show real sync date in "Fetch more messages" button

### PR DESCRIPTION
**Pull request by richard-ramos**
Requires
- https://github.com/status-im/status-lib/pull/123

### CODE REVIEW

**Review by jrainville — APPROVED** [review]

File: ui/imports/shared/controls/chat/FetchMoreMessagesButton.qml [hunk 1]
```diff
@@ -72,7 +72,8 @@ Item {
         horizontalAlignment: Text.AlignHCenter
         color: Style.current.secondaryText
         //% "before %1"
-        text: qsTrId("before--1").arg((nextMessageIndex > -1 ? new Date(nextMsgTimestamp * 1) : new Date()).toLocaleString(Qt.locale(localAppSettings.locale)))
```
Comment (alaibe) [inline ui/imports/shared/controls/chat/FetchMoreMessagesButton.qml:75]: Are the old attribute still in use?

**Review by alaibe — APPROVED** [review]

File: ui/imports/shared/controls/chat/FetchMoreMessagesButton.qml [hunk 2]
```diff
@@ -72,7 +72,8 @@ Item {
...
+        text: qsTrId("before--1").arg(new Date(chatsModel.channelView.activeChannel.syncedTo * 1000).toLocaleString(Qt.locale(localAppSettings.locale)))
```
Comment (borismelnik) [inline ui/imports/shared/controls/chat/FetchMoreMessagesButton.qml:75]:
```


string arg(value)

Returns a copy of this string with the lowest numbered place marker replaced by value, i.e., %1, %2, ..., %99. 
```
Sorry, but what `arg` this actually do? what expected?

File: ui/imports/shared/controls/chat/FetchMoreMessagesButton.qml (hunk 1 above)
Comment (richard-ramos) [reply ui/imports/shared/controls/chat/FetchMoreMessagesButton.qml:75]: I'll remove those!

File: ui/imports/shared/controls/chat/FetchMoreMessagesButton.qml (hunk 2 above)
Comment (richard-ramos) [reply ui/imports/shared/controls/chat/FetchMoreMessagesButton.qml:75]: `qsTrId("before--1")` ends up being translated to `"before %1"`, then the `.arg(...)` will replace `%1` by a date

**Pull request merged**